### ISSUE TITLE
ipptransform produces broken/empty output

### ISSUE THREAD

**Issue by attah** [OpenPrinting/libcups#76]
**Describe the bug**
When running ipptransform the resulting file comes out (visually) empty.

**To Reproduce**
1. ipptransform freshly built from source (using the submodule'd pdfio)
2. `LD_LIBRARY_PATH=./cups/ ./tools/ipptransform -m application/pdf ~/Downloads/cs-ippraster10-20120420-5102.4.pdf > out.pdf`
3. Open the resulting out.pdf - it will look empty and print errors about "Unknown operator" and "Too few (0) args to 're' operator"

**Expected behavior**
Output files show the same content (visually) as the input.

**System Information:**
 - OS: Linux mint 21.3
 - Browser: Chromium 120, Firefox 121
 - Version: ec3edbb3fc79fd1f7690ccc0f544fec06b8ba357

**Additional context**
XPDF manages to open/render the file (still printing the same errors), but anything Poppler-based, Firefox and Chromium all fail to show anything at all.
Poppler not being able to display the PDF also makes the issue propagate to raster formats if ipptransform uses Poppler pdftoppm.
I have this happening for all the handful of PDFs tested, but should i have been extraordinarily unlucky, use http://ftp.pwg.org/pub/pwg/candidates/cs-ippraster10-20120420-5102.4.pdf

The problem is not entirely new, but has been around a while now, i just didn't get around to reporting it.

I was suspecting locale float formatting issues, but my shell has $LANG en_US.UTF-8.

**Comment by michaelrsweet** [comment]
Please attach the output you are getting.

**Comment by attah** [comment]
[out.pdf](https://github.com/OpenPrinting/libcups/files/14043552/out.pdf)
Here you go.

**Comment by michaelrsweet** [comment]
Hmm, strange I'm not seeing this for current master but your output file is definitely damaged somehow... Can you try current master (I just tested 14e32422a)?

**Comment by attah** [comment]
That's just a one commit diff to where i was... but retested none the less. Same result.
(reinitialized the submodule, git-cleaned, ./configure'd and recompiled)
Double-checked the input file - still looks good.

**Comment by michaelrsweet** [comment]
Hmm, what processor, compiler, and glibc are you using? The locale definitely shouldn't be an issue...

**Comment by michaelrsweet** [comment]
Also, can you attach the stderr output from ipptransform (all the DEBUG: messages from ipptransform)?

**Comment by attah** [comment]
AMD Ryzen 9 3950X
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
Ubuntu GLIBC 2.35-0ubuntu3.6

```
attah@obsidian ~/repos/libcups $ LD_LIBRARY_PATH=./cups/ ./tools/ipptransform -m application/pdf ~/Downloads/cs-ippraster10-20120420-5102.4.pdf > out.pdf
DEBUG: page_dict_cb(dict=0x5556534333c0, key="ColorSpace", outpage=0x7ffc1eeaffa8), type=5
DEBUG: page_dict_cb(dict=0x5556534333c0, key="Font", outpage=0x7ffc1eeaffa8), type=5
DEBUG: page_dict_cb(dict=0x5556534333c0, key="ProcSet", outpage=0x7ffc1eeaffa8), type=1
DEBUG: page_dict_cb(dict=0x5556534333c0, key="XObject", outpage=0x7ffc1eeaffa8), type=5
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565343b360, key="ColorSpace", outpage=0x7ffc1eeb00d8), type=5
DEBUG: page_dict_cb(dict=0x55565343b360, key="Font", outpage=0x7ffc1eeb00d8), type=5
DEBUG: page_dict_cb(dict=0x55565343b360, key="ProcSet", outpage=0x7ffc1eeb00d8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653473100, key="ColorSpace", outpage=0x7ffc1eeb0208), type=5
DEBUG: page_dict_cb(dict=0x555653473100, key="Font", outpage=0x7ffc1eeb0208), type=5
DEBUG: page_dict_cb(dict=0x555653473100, key="ProcSet", outpage=0x7ffc1eeb0208), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565346e6b0, key="ColorSpace", outpage=0x7ffc1eeb0338), type=5
DEBUG: page_dict_cb(dict=0x55565346e6b0, key="Font", outpage=0x7ffc1eeb0338), type=5
DEBUG: page_dict_cb(dict=0x55565346e6b0, key="ProcSet", outpage=0x7ffc1eeb0338), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565346f420, key="ColorSpace", outpage=0x7ffc1eeb0468), type=5
DEBUG: page_dict_cb(dict=0x55565346f420, key="Font", outpage=0x7ffc1eeb0468), type=5
DEBUG: page_dict_cb(dict=0x55565346f420, key="ProcSet", outpage=0x7ffc1eeb0468), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653472250, key="ColorSpace", outpage=0x7ffc1eeb0598), type=5
DEBUG: page_dict_cb(dict=0x555653472250, key="Font", outpage=0x7ffc1eeb0598), type=5
DEBUG: page_dict_cb(dict=0x555653472250, key="ProcSet", outpage=0x7ffc1eeb0598), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653475df0, key="ColorSpace", outpage=0x7ffc1eeb06c8), type=5
DEBUG: page_dict_cb(dict=0x555653475df0, key="Font", outpage=0x7ffc1eeb06c8), type=5
DEBUG: page_dict_cb(dict=0x555653475df0, key="ProcSet", outpage=0x7ffc1eeb06c8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653476b40, key="ColorSpace", outpage=0x7ffc1eeb07f8), type=5
DEBUG: page_dict_cb(dict=0x555653476b40, key="Font", outpage=0x7ffc1eeb07f8), type=5
DEBUG: page_dict_cb(dict=0x555653476b40, key="ProcSet", outpage=0x7ffc1eeb07f8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653479830, key="ColorSpace", outpage=0x7ffc1eeb0928), type=5
DEBUG: page_dict_cb(dict=0x555653479830, key="Font", outpage=0x7ffc1eeb0928), type=5
DEBUG: page_dict_cb(dict=0x555653479830, key="ProcSet", outpage=0x7ffc1eeb0928), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565347f250, key="ColorSpace", outpage=0x7ffc1eeb0a58), type=5
DEBUG: page_dict_cb(dict=0x55565347f250, key="Font", outpage=0x7ffc1eeb0a58), type=5
DEBUG: page_dict_cb(dict=0x55565347f250, key="ProcSet", outpage=0x7ffc1eeb0a58), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565347e4e0, key="ColorSpace", outpage=0x7ffc1eeb0b88), type=5
DEBUG: page_dict_cb(dict=0x55565347e4e0, key="Font", outpage=0x7ffc1eeb0b88), type=5
DEBUG: page_dict_cb(dict=0x55565347e4e0, key="ProcSet", outpage=0x7ffc1eeb0b88), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565347d2c0, key="ColorSpace", outpage=0x7ffc1eeb0cb8), type=5
DEBUG: page_dict_cb(dict=0x55565347d2c0, key="Font", outpage=0x7ffc1eeb0cb8), type=5
DEBUG: page_dict_cb(dict=0x55565347d2c0, key="ProcSet", outpage=0x7ffc1eeb0cb8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565347c310, key="ColorSpace", outpage=0x7ffc1eeb0de8), type=5
DEBUG: page_dict_cb(dict=0x55565347c310, key="Font", outpage=0x7ffc1eeb0de8), type=5
DEBUG: page_dict_cb(dict=0x55565347c310, key="ProcSet", outpage=0x7ffc1eeb0de8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534632d0, key="ColorSpace", outpage=0x7ffc1eeb0f18), type=5
DEBUG: page_dict_cb(dict=0x5556534632d0, key="Font", outpage=0x7ffc1eeb0f18), type=5
DEBUG: page_dict_cb(dict=0x5556534632d0, key="ProcSet", outpage=0x7ffc1eeb0f18), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565345ea50, key="ColorSpace", outpage=0x7ffc1eeb1048), type=5
DEBUG: page_dict_cb(dict=0x55565345ea50, key="Font", outpage=0x7ffc1eeb1048), type=5
DEBUG: page_dict_cb(dict=0x55565345ea50, key="ProcSet", outpage=0x7ffc1eeb1048), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653461c30, key="ColorSpace", outpage=0x7ffc1eeb1178), type=5
DEBUG: page_dict_cb(dict=0x555653461c30, key="Font", outpage=0x7ffc1eeb1178), type=5
DEBUG: page_dict_cb(dict=0x555653461c30, key="ProcSet", outpage=0x7ffc1eeb1178), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653460200, key="ColorSpace", outpage=0x7ffc1eeb12a8), type=5
DEBUG: page_dict_cb(dict=0x555653460200, key="Font", outpage=0x7ffc1eeb12a8), type=5
DEBUG: page_dict_cb(dict=0x555653460200, key="ProcSet", outpage=0x7ffc1eeb12a8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534642f0, key="ColorSpace", outpage=0x7ffc1eeb13d8), type=5
DEBUG: page_dict_cb(dict=0x5556534642f0, key="Font", outpage=0x7ffc1eeb13d8), type=5
DEBUG: page_dict_cb(dict=0x5556534642f0, key="ProcSet", outpage=0x7ffc1eeb13d8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565346a070, key="ColorSpace", outpage=0x7ffc1eeb1508), type=5
DEBUG: page_dict_cb(dict=0x55565346a070, key="Font", outpage=0x7ffc1eeb1508), type=5
DEBUG: page_dict_cb(dict=0x55565346a070, key="ProcSet", outpage=0x7ffc1eeb1508), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565346b610, key="ColorSpace", outpage=0x7ffc1eeb1638), type=5
DEBUG: page_dict_cb(dict=0x55565346b610, key="Font", outpage=0x7ffc1eeb1638), type=5
DEBUG: page_dict_cb(dict=0x55565346b610, key="ProcSet", outpage=0x7ffc1eeb1638), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565346c9c0, key="ColorSpace", outpage=0x7ffc1eeb1768), type=5
DEBUG: page_dict_cb(dict=0x55565346c9c0, key="Font", outpage=0x7ffc1eeb1768), type=5
DEBUG: page_dict_cb(dict=0x55565346c9c0, key="ProcSet", outpage=0x7ffc1eeb1768), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565346dd80, key="ColorSpace", outpage=0x7ffc1eeb1898), type=5
DEBUG: page_dict_cb(dict=0x55565346dd80, key="Font", outpage=0x7ffc1eeb1898), type=5
DEBUG: page_dict_cb(dict=0x55565346dd80, key="ProcSet", outpage=0x7ffc1eeb1898), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534692a0, key="ColorSpace", outpage=0x7ffc1eeb19c8), type=5
DEBUG: page_dict_cb(dict=0x5556534692a0, key="Font", outpage=0x7ffc1eeb19c8), type=5
DEBUG: page_dict_cb(dict=0x5556534692a0, key="ProcSet", outpage=0x7ffc1eeb19c8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653469a90, key="ColorSpace", outpage=0x7ffc1eeb1af8), type=5
DEBUG: page_dict_cb(dict=0x555653469a90, key="Font", outpage=0x7ffc1eeb1af8), type=5
DEBUG: page_dict_cb(dict=0x555653469a90, key="ProcSet", outpage=0x7ffc1eeb1af8), type=1
DEBUG: page_dict_cb(dict=0x555653469a90, key="XObject", outpage=0x7ffc1eeb1af8), type=5
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534664d0, key="ColorSpace", outpage=0x7ffc1eeb1c28), type=5
DEBUG: page_dict_cb(dict=0x5556534664d0, key="Font", outpage=0x7ffc1eeb1c28), type=5
DEBUG: page_dict_cb(dict=0x5556534664d0, key="ProcSet", outpage=0x7ffc1eeb1c28), type=1
DEBUG: page_dict_cb(dict=0x5556534664d0, key="XObject", outpage=0x7ffc1eeb1c28), type=5
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565345ba00, key="ColorSpace", outpage=0x7ffc1eeb1d58), type=5
DEBUG: page_dict_cb(dict=0x55565345ba00, key="Font", outpage=0x7ffc1eeb1d58), type=5
DEBUG: page_dict_cb(dict=0x55565345ba00, key="ProcSet", outpage=0x7ffc1eeb1d58), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653454670, key="ColorSpace", outpage=0x7ffc1eeb1e88), type=5
DEBUG: page_dict_cb(dict=0x555653454670, key="Font", outpage=0x7ffc1eeb1e88), type=5
DEBUG: page_dict_cb(dict=0x555653454670, key="ProcSet", outpage=0x7ffc1eeb1e88), type=1
DEBUG: page_dict_cb(dict=0x555653454670, key="XObject", outpage=0x7ffc1eeb1e88), type=5
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565345a1d0, key="ColorSpace", outpage=0x7ffc1eeb1fb8), type=5
DEBUG: page_dict_cb(dict=0x55565345a1d0, key="Font", outpage=0x7ffc1eeb1fb8), type=5
DEBUG: page_dict_cb(dict=0x55565345a1d0, key="ProcSet", outpage=0x7ffc1eeb1fb8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565345cbb0, key="ColorSpace", outpage=0x7ffc1eeb20e8), type=5
DEBUG: page_dict_cb(dict=0x55565345cbb0, key="Font", outpage=0x7ffc1eeb20e8), type=5
DEBUG: page_dict_cb(dict=0x55565345cbb0, key="ProcSet", outpage=0x7ffc1eeb20e8), type=1
DEBUG: page_dict_cb(dict=0x55565345cbb0, key="XObject", outpage=0x7ffc1eeb20e8), type=5
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653458dc0, key="ColorSpace", outpage=0x7ffc1eeb2218), type=5
DEBUG: page_dict_cb(dict=0x555653458dc0, key="Font", outpage=0x7ffc1eeb2218), type=5
DEBUG: page_dict_cb(dict=0x555653458dc0, key="ProcSet", outpage=0x7ffc1eeb2218), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565343fe00, key="ColorSpace", outpage=0x7ffc1eeb2348), type=5
DEBUG: page_dict_cb(dict=0x55565343fe00, key="Font", outpage=0x7ffc1eeb2348), type=5
DEBUG: page_dict_cb(dict=0x55565343fe00, key="ProcSet", outpage=0x7ffc1eeb2348), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534447f0, key="ColorSpace", outpage=0x7ffc1eeb2478), type=5
DEBUG: page_dict_cb(dict=0x5556534447f0, key="Font", outpage=0x7ffc1eeb2478), type=5
DEBUG: page_dict_cb(dict=0x5556534447f0, key="ProcSet", outpage=0x7ffc1eeb2478), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534459e0, key="ColorSpace", outpage=0x7ffc1eeb25a8), type=5
DEBUG: page_dict_cb(dict=0x5556534459e0, key="Font", outpage=0x7ffc1eeb25a8), type=5
DEBUG: page_dict_cb(dict=0x5556534459e0, key="ProcSet", outpage=0x7ffc1eeb25a8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534409d0, key="ColorSpace", outpage=0x7ffc1eeb26d8), type=5
DEBUG: page_dict_cb(dict=0x5556534409d0, key="Font", outpage=0x7ffc1eeb26d8), type=5
DEBUG: page_dict_cb(dict=0x5556534409d0, key="ProcSet", outpage=0x7ffc1eeb26d8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653443400, key="ColorSpace", outpage=0x7ffc1eeb2808), type=5
DEBUG: page_dict_cb(dict=0x555653443400, key="Font", outpage=0x7ffc1eeb2808), type=5
DEBUG: page_dict_cb(dict=0x555653443400, key="ProcSet", outpage=0x7ffc1eeb2808), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x555653446080, key="ColorSpace", outpage=0x7ffc1eeb2938), type=5
DEBUG: page_dict_cb(dict=0x555653446080, key="Font", outpage=0x7ffc1eeb2938), type=5
DEBUG: page_dict_cb(dict=0x555653446080, key="ProcSet", outpage=0x7ffc1eeb2938), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534478a0, key="ColorSpace", outpage=0x7ffc1eeb2a68), type=5
DEBUG: page_dict_cb(dict=0x5556534478a0, key="Font", outpage=0x7ffc1eeb2a68), type=5
DEBUG: page_dict_cb(dict=0x5556534478a0, key="ProcSet", outpage=0x7ffc1eeb2a68), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x5556534488c0, key="ColorSpace", outpage=0x7ffc1eeb2b98), type=5
DEBUG: page_dict_cb(dict=0x5556534488c0, key="Font", outpage=0x7ffc1eeb2b98), type=5
DEBUG: page_dict_cb(dict=0x5556534488c0, key="ProcSet", outpage=0x7ffc1eeb2b98), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565344cc60, key="ColorSpace", outpage=0x7ffc1eeb2cc8), type=5
DEBUG: page_dict_cb(dict=0x55565344cc60, key="Font", outpage=0x7ffc1eeb2cc8), type=5
DEBUG: page_dict_cb(dict=0x55565344cc60, key="ProcSet", outpage=0x7ffc1eeb2cc8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
DEBUG: page_dict_cb(dict=0x55565344a6f0, key="ColorSpace", outpage=0x7ffc1eeb2df8), type=5
DEBUG: page_dict_cb(dict=0x55565344a6f0, key="Font", outpage=0x7ffc1eeb2df8), type=5
DEBUG: page_dict_cb(dict=0x55565344a6f0, key="ProcSet", outpage=0x7ffc1eeb2df8), type=1
DEBUG: iwidth=612, iheight=792, cwidth=559,843, cheight=771,024, rotate=false
DEBUG: Opening content stream 1/1...
DEBUG: Opened stream 1, resmap[0]=(nil)
```

**Comment by michaelrsweet** [comment]
The output indicates a different locale than `en_US.UTF-8`. How about attaching the output of "env"?

This will likely require some changes to force the `LC_NUMERIC` locale category to "C" when reading/writing PDF files (or update PDFio to go through the same hoops that libcups already does to work around the locale stuff), but I'd like to confirm what is happening on your system...

**Comment by attah** [comment]
Seems my hunch was right then. I just wasn't aware there were multiple environment variables for this.
Running with `LC_NUMERIC=C` indeed fixes the issue. (LC_NUMERIC=sv_SE.UTF-8 is what i have normally).

**Comment by michaelrsweet** [comment]
OK, thanks for confirming. I've opened a [PDFio issue](https://github.com/michaelrsweet/pdfio/issues/61) to track the fix, and will update this bug once we update the pdfio submodule in libcups.

**Comment by michaelrsweet** [comment]
OK, fix for PDFio is pushed...

[master 886726b0a] Pickup locale fix for PDFio (Issue #76)

**Comment by attah** [comment]
For completeness; this does indeed fix the issue. Thanks!
